provider/pkg/provider/helm_release.go

```diff
@@ -483,7 +580,11 @@ func (r *helmReleaseProvider) helmCreate(ctx context.Context, urn resource.URN,
 	client.WaitForJobs = !newRelease.SkipAwait && newRelease.WaitForJobs
 	client.Devel = newRelease.Devel
 	client.DependencyUpdate = newRelease.DependencyUpdate
-	client.Timeout = time.Duration(newRelease.Timeout) * time.Second
+	timeout := newRelease.Timeout
```

Comment:
It looks like this logic occurs three times in this file. Can we refactor it out into a function, eg. `timeoutOrDefault` or something like that?

Reply:
➕